src/index.html

```diff
@@ -16,7 +16,11 @@
       href="./styles/index.scss"
```

Comment:
Browsers do not directly support SCSS files. Ensure that `index.scss` is compiled to a CSS file and linked appropriately in the HTML. You might want to use a tool like Sass to compile your SCSS to CSS.